contracts/SushiYieldSource.sol

```diff
@@ -45,10 +45,12 @@ contract SushiYieldSource is IYieldSource {
     /// @param amount The amount of `token()` to be supplied
     /// @param to The user whose balance will receive the tokens
     function supplyTokenTo(uint256 amount, address to) public override {
-        sushiAddr.transferFrom(msg.sender, address(this), amount);
-        sushiAddr.approve(address(sushiBar), amount);
-
+        ISushi sushi = sushiAddr;
         ISushiBar bar = sushiBar;
+
+        sushi.transferFrom(msg.sender, address(this), amount);
+        sushi.approve(address(bar), amount);
```

Comment:
Make sure when you merge to include your safe pr! https://github.com/pooltogether/sushi-pooltogether/pull/11